docs/log-correlation.asciidoc

```diff
@@ -94,9 +94,9 @@ This would print a log message along the lines of:
 
     2019/09/17 14:48:02 ERROR [trace.id=cd04f33b9c0c35ae8abe77e799f126b7 transaction.id=cd04f33b9c0c35ae span.id=960834f4538880a4] an error occurred
 
-For log correlation to work, the trace IDs must be extracted from the log message and
+For the log correlation to work, the trace IDs must be extracted from the log message and
 stored in separate fields in the Elasticsearch document. This can be achieved by
-{filebeat-ref}/configuring-ingest-node.html[parsing the data by using ingest node], in particular
+{filebeat-ref}/configuring-ingest-node.html[parsing the data using an ingest node], in particular
```

Comment:
```suggestion
{filebeat-ref}/configuring-ingest-node.html[using an ingest pipeline to parse the data], in particular
```

Reply:
I'd change the link. The topic title puts the parsing first to emphasize it, but you don't have to do that in the link.
Also I think we are getting away from talking about "ingest node" and instead using "ingest pipeline" because it's more intuitive.